.github/workflows/automated-version-update-pr.yml

```diff
@@ -3,12 +3,18 @@ on:
     inputs:
       gutenbergMobileVersion:
         required: true
+      title:
+        required: false
+      prURL:
+        required: false
 
 jobs:
   create-pull-request:
     runs-on: ubuntu-latest
     env:
       GUTENBERG_MOBILE_VERSION: ${{ github.event.inputs.gutenbergMobileVersion }}
+      PR_TITLE: ${{ github.event.inputs.title }}
```

Comment:
Could we do something like `PR_TITLE: ${{ github.event.inputs.title || "Automated gutenberg-mobile version update"}}` with GH expression syntax?